src/server/physics/map-builder.js

```diff
@@ -0,0 +1,96 @@
+module.exports = function(Ammo, world, map) {
+	let _startAreas = [],
+		_endAreas = [];
+
+	map.currentMapData.then((map) => {
+		function transformFromEntity(entity) {
```

Comment:
I think it would be cleaner to make this a private function instead of defining it here.